certbot/plugins/manual.py

```diff
@@ -183,15 +211,30 @@ def _perform_single(self, achall):
 
         return response
 
-    def _notify_and_wait(self, message):  # pylint: disable=no-self-use
-        # TODO: IDisplay wraps messages, breaking the command
-        #answer = zope.component.getUtility(interfaces.IDisplay).notification(
-        #    message=message, height=25, pause=True)
-        sys.stdout.write(message)
-        six.moves.input("Press ENTER to continue")
+    def _perform_dns01_challenge(self, achall):
+        response, validation = achall.response_and_validation()
+        if not self.conf("test-mode"):
+            if not self.conf("public-ip-logging-ok"):
+                if not zope.component.getUtility(interfaces.IDisplay).yesno(
```

Comment:
Slightly  unrelated to this PR, but If I run this with two domains, it asks for permission twice. Maybe we should just add an `else` case, like this:
```
else:
    self.config.namespace.manual_public_ip_logging_ok = True
```
So that it only asks once?